pom.xml

```diff
@@ -404,14 +403,15 @@
 									<ignoredUsedUndeclaredDependency>io.netty:netty-codec-http</ignoredUsedUndeclaredDependency>
 									<ignoredUsedUndeclaredDependency>io.netty:netty-common</ignoredUsedUndeclaredDependency>
 									<ignoredUsedUndeclaredDependency>jakarta.annotation:jakarta.annotation-api</ignoredUsedUndeclaredDependency>
-									<ignoredUsedUndeclaredDependency>org.quartz-scheduler:quartz</ignoredUsedUndeclaredDependency>
+									<ignoredUsedUndeclaredDependency>io.quarkus:quarkus-scheduler-api</ignoredUsedUndeclaredDependency>
+									<ignoredUsedUndeclaredDependency>com.fasterxml.jackson.core:jackson-annotations</ignoredUsedUndeclaredDependency>
```

Comment:
nothing changed about jackson annotations, interesting to know why it was needed.

Reply:
It happened after I changed the `sourcePath` in the `javadoc` plugin if my memory is right therefore I had to use this